### ISSUE TITLE
Verify that RNW works with Node 14

### ISSUE THREAD

**Issue by acoates-ms** [microsoft/react-native-windows#5725]
We should run through basic dev scenarios and verify that things work with node 14.

**Comment by asklar** [comment]
lots of warnings when running `yarn windows`, they seem innocent but make the output hard to parse:

```
(node:5928) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5928) Warning: Accessing non-existent property 'cd' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'chmod' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'cp' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'dirs' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'pushd' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'popd' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'echo' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'tempdir' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'pwd' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'exec' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'ls' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'find' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'grep' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'head' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'ln' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'mkdir' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'rm' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'mv' of module exports inside circular dependency                  
(node:5928) Warning: Accessing non-existent property 'sed' of module exports inside circular dependency                 
(node:5928) Warning: Accessing non-existent property 'set' of module exports inside circular dependency                 
(node:5928) Warning: Accessing non-existent property 'sort' of module exports inside circular dependency               
(node:5928) Warning: Accessing non-existent property 'tail' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'test' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'to' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'toEnd' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'touch' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'uniq' of module exports inside circular dependency
(node:5928) Warning: Accessing non-existent property 'which' of module exports inside circular dependency
```

**Comment by NickGerleman** [comment]
I fixed the warnings and have been using Node 14 for local development for a while. Haven't noticed any issues with day-to-dat functionality, so I'm going to close this unless we find specific issues.
